modules/account-lib/src/mpc/tss.ts

```diff
@@ -387,12 +387,8 @@ export default class Eddsa {
   }
 
   verify(message: Buffer, signature: Signature): Buffer {
-    const publicKey = bigIntFromBufferLE(Buffer.from(signature.y, 'hex'));
-    const signedMessage = Buffer.concat([
-      Buffer.from(signature.R, 'hex'),
-      Buffer.from(signature.sigma, 'hex'),
-      message,
-    ]);
-    return Eddsa.curve.verify(publicKey, signedMessage);
+    const publicKey = Buffer.from(signature.y, 'hex');
+    const signedMessage = Buffer.concat([Buffer.from(signature.R, 'hex'), Buffer.from(signature.sigma, 'hex')]);
+    return Eddsa.curve.verify(message, signedMessage, publicKey);
```

Comment:
This seems specific to eddsa curve

Reply:
Will be refactoring TSS file in another PR to address this